services/backend/scripts/install_localstack_fixtures.sh

```diff
@@ -14,9 +14,25 @@ function wait_for_s3 {
   done
 }
 
-
+function is_event_bus_exists {
+  out=$(
+    aws --region=eu-west-1 --endpoint-url="$LOCALSTACK_URL" events list-event-buses | jq ".EventBuses[] | select(.Name == \"$1\") | .Name"
+  )
+  if [[ -z "$out" ]]; then
+    return 1
+  else
+    return 0
+  fi
```

Comment:
You could do `return [[ -n "$out" ]]`, I think.

Reply:
`./scripts/install_localstack_fixtures.sh: line 21: return: [: numeric argument required` but I think we could replace with:
`test -n "$out"
  return $?`
What do you think @Toreno96 ?

Reply:
Let's stay w/ previous version then, it's more readable IMHO.

Reply:
done :)